src/Features/Kanban.php

```diff
@@ -176,4 +180,20 @@ public function getKanbanUrlWithID(int $items_id, bool $full = true): string
         }
         return static::getFormURLWithID($items_id, $full) . "&forcetab={$tab_id}";
     }
+
+    public static function getTeamMemberForm(CommonDBTM $item)
+    {
+        $members_types = ProjectTeam::$available_types;
+
+        TemplateRenderer::getInstance()->display('components/kanban/teammember.html.twig', [
+            'item' => $item,
+            'entities_id' => 0,
+            'canupdate' => true,
+            'canassign' => true,
+            'params' => $item->fields + ['load_actors' => false],
+            'members_types' => $members_types,
+            'canassigntome' => false,
+            'allow_auto_submit' => false,
```

Comment:
Many of these variables are not used in the `components/kanban/teammember.html.twig` template.